packages/playground/src/routes/workflows.new.tsx

```diff
@@ -21,36 +22,36 @@ export function NewWorkflow() {
     console.error("No OpenAPI spec found");
   }
 
-  const handleSubmit = () => {
-    createWorkflow({
+  const handleSubmit = useHandler(() => {
```

Comment:
shouldn't we recreate this callback when user story or (unlikely) open api schema changes?